### PR TITLE
added StartScreen and fixed uploading functions

### PR DESCRIPTION
**Pull request by ricardomendez121**

### CODE REVIEW

File: lib/models/media_type.dart [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  MediaType(
+      {required this.index,
+      required this.title,
+      required this.subTitle,
+      required this.iconName});
+
+  static List<MediaType> listTypes = <MediaType>[
+    MediaType(
+        index: 0,
+        title: 'Image',
```
Comment (haybankz) [inline lib/models/media_type.dart]: Move string literals to the constant file

File: lib/screens/home_screen.dart [hunk 2]
```diff
@@ -32,10 +39,24 @@ class _HomeScreenState extends State<HomeScreen> {
...
 
-  List title = ["Upload", "Description", "Mint", "Publish"];
+  void setCurrentPage(page) {
+    setState(() {
+      _page = page;
+    });
+  }
+
+  void setMediaType(_type) {
+    setState(() {
```
Comment (haybankz) [inline lib/screens/home_screen.dart]: use valueNotifier instead of setState

File: lib/screens/home_screen.dart [hunk 3]
```diff
@@ -67,91 +88,95 @@ class _HomeScreenState extends State<HomeScreen> {
...
+                          ),
+                        ],
+                      ),
                     ),
-                  ],
-                ),
-              ),
-            ),
+                  )
+                : Container(),
```
Comment (haybankz) [inline lib/screens/home_screen.dart]: use const SizedBox() instead

File: lib/screens/routing_screen.dart [hunk 4]
```diff
@@ -23,15 +23,20 @@ class _RoutingScreenState extends State<RoutingScreen> {
 
   @override
   void initState() {
-    super.initState();
+    super.initState(); 
+    Future.delayed(const Duration(seconds: 1), () {
+      PylonsWallet.instance.exists().then((walletExists) async {
```
Comment (haybankz) [inline lib/screens/routing_screen.dart:28]: this is a repetition of the code line 34-36, is there a reason why we are checking if the wallet app exists twice?

File: lib/screens/start_screen.dart [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+
+class _UploadWidgetState extends State<_UploadWidget> {
+  @override
+  Widget build(BuildContext context) {
+    final sizeWidth = MediaQuery.of(context).size.width;
+    return Consumer<EaselProvider>(
+      builder: (_, provider, __) => Column(
+        children: [
+          Text(
+            "Choose your NFT format",
```
Comment (haybankz) [inline lib/screens/start_screen.dart]: move string to the constant file

File: lib/screens/start_screen.dart [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+            width: screenSize.width(),
+            child: ClipRRect(
+              borderRadius: const BorderRadius.only(
+                  topLeft: Radius.circular(14),
+                  bottomLeft: Radius.circular(14)),
+              child: BackdropFilter(
+                filter: ImageFilter.blur(sigmaX: 2.0, sigmaY: 2.0),
+                child: Container(
+                  height: screenSize.height(percent: 85),
+                  // width: screenSize.width(),
```
Comment (haybankz) [inline lib/screens/start_screen.dart]: remove dead code

**Review by haybankz — CHANGES_REQUESTED** [review]

**Review by haybankz — COMMENTED** [review]
Please make requested changes.

**Review by kjawadDeveloper2 — APPROVED** [review]
Looks good to me.

**Review by starhelios — APPROVED** [review]

**Pull request merged**